### PR TITLE
AB#428101: Feature/428101 refactor models for multiple sheets

### PR DESCRIPTION
**Pull request by hanna1013**
# Pull Request Details

## What this PR does / why we need it:
Refactored the models to allow multiple sheets to be parsed. Additional tests added.

_A brief description of changes being made_
 Relates to ADO Work Item [AB#428101](https://dev.azure.com/defragovuk/0007dd9e-c7cf-473a-a859-4aad7bd32c5e/_workitems/edit/428101)

## Special notes for your reviewer

_Any specific actions or notes on review?_

## Testing

_Any relevant testing information and pipeline runs._

## Checklist (please delete before completing or setting auto-complete)

- [ ] Story Work items associated (not Tasks)
- [ ] Successful testing run(s) link provided
- [ ] Title pattern should be `{work item number}: {title}`
- [ ] Description covers all the changes in the PR
- [ ] This PR contains documentation
- [ ] This PR contains tests
- [ ] If the PR includes a new parser, please ensure the happy & unhappy path test files are linked to [this](https://eaflood.atlassian.net/wiki/spaces/EX/pages/4855562467/Packing+List+analysis#Top-20-Traders) page and the detailed description is added to [this](https://eaflood.atlassian.net/wiki/spaces/EX/pages/4855562467/Packing+List+analysis) page. Thanks

## How does this PR make you feel:

gif

### CODE REVIEW

File: app/services/parsers/bandm/model1.js [hunk 1]
```diff
@@ -1,43 +1,52 @@
-const CombineParser = require("../../parser-combine");
-const ParserModel = require("../../parser-model");
+const combine_parser = require("../../parser-combine");
```
Comment (david-pizzi-cg) [inline app/services/parsers/bandm/model1.js:1]: this change triggers issue for local naming issue on [SonarCloud](https://sonarcloud.io/project/issues?issueStatuses=OPEN%2CCONFIRMED&sinceLeakPeriod=true&branch=feature%2F428101-refactor-models-for-multiple-sheets&id=trade-exportscore-plp)

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "trade-exportscore-plp",
-  "version": "4.0.0",
+  "version": "4.0.2",
```
Comment (david-pizzi-cg) [inline package.json:3]: why not 4.0.1?

Comment (hanna1013) [reply package.json:3]: ahh mistake, I'll change it now

Comment (hanna1013) [reply package.json:3]: I can't change it, so I'll have to leave it as 4.0.2

**Review by david-pizzi-cg — APPROVED** [review]

**Pull request merged**